Packs/PAN-OS/Playbooks/playbook-PAN-OS_Log_Forwarding_Setup_And_Maintenance_README.md

```diff
@@ -1,6 +1,13 @@
 Sets up and maintains log forwarding for the Panorama rulebase.
 It can be run when setting up a new instance, or as a periodic job to enforce log forwarding policy.
 You can either update all rules and override previous profiles, or update only rules that do not have a log forwarding profile configured.
+​
+## Description & Playbook Flow
+---
+The playbook lists all of the rules (either pre-rulebase or post-rulebase). 
+If the analyst wishes to override log-forwarding profile for all rules, it updates all the rules with the given profile.
+If the analyst wishes to update only rules with missing profile, it will update only those rules.
+in the end the playbook will commit to PAN-OS (either auto commit or manual commit by the analyst)
```

Comment:
```suggestion
If the analyst wishes to override the log-forwarding profile for all rules, it updates all the rules with the given profile.
If the analyst wishes to update only rules with missing profiles, it will update only those rules.
In the end, the playbook will commit to PAN-OS (either auto-commit or manual commit by the analyst).
```